### PR TITLE
[reproducer] add doc note to create CRC with proper size

### PR DESCRIPTION
**Pull request by stuggi**
As a pull request owner and reviewers, I checked that:
- [x] Appropriate testing is done and actually running
- [x] Appropriate documentation exists and/or is up-to-date:
  - [x] README in the role
  - [x] Content of the docs/source is reflecting the changes

### CODE REVIEW

File: docs/source/reproducers/01-considerations.md [hunk 1]
```diff
@@ -14,6 +14,9 @@ The CRC VM has to be properly set up prior to running the reproducer play. You
 may get to that leveraging either the rhol_crc role from the CI Framework, or
 manually by calling `crc setup && crc start -p pull-secret`.
 
+Note: make sure to create the CRC with proper disk size it won't be resized
```
Comment (cjeanner) [inline docs/source/reproducers/01-considerations.md:17]: maybe provide a reasonable disk size? IIRC we usually point to 100G ?

Comment (stuggi) [reply docs/source/reproducers/01-considerations.md]: done

**Pull request merged**